src/script/arch-update.sh

```diff
@@ -189,33 +189,18 @@ invalid_option() {
 	exit 1
 }
 
-# Definition of the icon directory
-icon_dir="/usr/share/icons/hicolor/scalable/apps"
-
-# Definition of the state_checking function: Change state to "checking"
-state_checking() {
-	echo "${name}_checking" > "${statedir}/current_state"
-}
-
 # Definition of the state_updates_available function: Change state to "updates-available"
 state_updates_available() {
 	echo "${name}_updates-available" > "${statedir}/current_state"
 }
 
-# Definition of the state_installing function: Change state to "installing"
-state_installing() {
-	echo "${name}_installing" > "${statedir}/current_state"
-}
-
 # Definition of the state_up_to_date function: Change state to "up to date"
 state_up_to_date() {
 	echo "${name}_up-to-date" > "${statedir}/current_state"
```

Comment:
@trigg Since the `arch-update.svg` icon and the `arch-update_up-to-date.svg` should be the same, it is fine to simply `echo ${name}` here?
Unless it would break something regarding the applet, I guess there's no need to keep 2 icons that are actually the same ^^

Reply:
`${name}` here would work fine.

Reply:
Alright, pushed in https://github.com/Antiz96/arch-update/pull/150/commits/5bdbd7375eae8adde69522439334fb7ad65dd16f